Chapter_3/ex3_floating.rb

```diff
@@ -0,0 +1,40 @@
+# This line prints a string
+puts 'I will now count my chickens:'
+
+# This line prints 'Hens' and interpolates the equation, printing 30
+puts "Hens #{25.0 + 30.0 / 6.0}"
```

Comment:
Just FYI... If any of the operators are floating point, the whole thing is evaluated with floating point operations.  i.e. `25 + 30 / 6.0` is the same thing.